back-end/helpers/put_star_interactor.js

```diff
@@ -0,0 +1,26 @@
+import {InputBoundaryInterface} from "../interfaces/input-boundary-interface.js";
+import OutputDataBoundary from "./output_data_boundary.js";
+import { IntentInterface } from "../interfaces/intent-interface.js";
+
+export default class PostTranscriptInteractor extends InputBoundaryInterface{
```

Comment:
class name is wrong

Reply:
It's fine I added them to the API call. I think I was not clear when explaining which parts we need or don't. But we do need project id , question and star status.